.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: generic
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker pull fitiotlab/iot-lab-gateway:latest
+  - docker build -t iot-lab-gateway .
+  - docker build -t iot-lab-gateway-test tests
+
+script:
+  - >
+    docker run -v gateway_code:/home/iot-lab-gateway/gateway_code
+    -v $CWD/tests_utils/cfg_dir:/var/local/config
+    -e BOARD_TYPE=m3
+    -e CONTROL_NODE_TYPE=no
+    -e HOSTNAME=m3-00 
+    -e IOTLAB_GATEWAY_NO_INTEGRATION_TESTS
+    --privileged iot-lab-gateway-test
+
+
+after_success:
```

Comment:
Should this be part of this PR ? I don't think so. Please wait after we are done with setting up travis. Otherwise, it will become a mess.

Reply:
ok. I think if the latest build is green this could be merged and close

Reply:
Yes, that's what I wanted to do (just testing locally with arduino zero right now)